mne/tests/test_what.py

```diff
@@ -0,0 +1,40 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+# License: BSD
+
+import glob
+import os.path as op
+
+import pytest
+
+from mne import what
+from mne.utils import run_tests_if_main
+from mne.datasets import testing
+
+data_path = testing.data_path(download=False)
+
+
+@pytest.mark.slowtest
+@testing.requires_testing_data
+def test_what():
```

Comment:
if the what function is written in io/base.py it should be tested in mne/io/tests/test_base.py